app/views/shared/_accounts_table.html.haml

```diff
@@ -7,16 +7,10 @@
     - @accounts.each do |account|
       %tr
         %td
-          - case
-          - when !current_ability.can?(:manage, account)
-            = account
-
-          - when current_facility.cross_facility?
+          - if current_ability.can?(:manage, account)
             = link_to account, facility_account_path(current_facility, account)
-
           - else
-            = link_to account,
```

Comment:
Was it accessible before the account administrator role was added? I feel like it went to a statement view. I think adding the tab sounds like a reasonable plan.